peep.py

```diff
@@ -920,7 +920,11 @@ def peep_port(paths):
         if not hashes:
             print(req.req)
         else:
-            print('%s' % req.req, end='')
+            if req.original_link:
```

Comment:
IIRC, I added the `original_link` attr only in pip 8, so this will crash in older versions. Also `req_string` isn't a string in the case of `req.req`. One (kind of cheap) way we could fix both is by replacing this whole `if` with an inline ternary.